lms/templates/ccx/enrollment.html

```diff
@@ -83,7 +83,30 @@
       <div class="bottom-bar">
         <label for="student-id" class="sr">${_("Enter username or email")}</label>
         <input name="student-id" id="student-id" class="add-field" placeholder="${_("Enter username or email")}" type="text">
-        <input name="student-action" class="add" value="Add Student" type="button">
+        <input name="student-action" class="add add-student-btn" value="Add Student" type="button">
+        <div class="enroll-option">
+            <input type="checkbox" name="auto-enroll" id="auto-enroll" value="Auto-Enroll" checked="yes" aria-describedby="auto-enroll-helper" disabled>
+            <label style="display:inline" for="auto-enroll">${_("Auto Enroll")}</label>
+            <div class="hint auto-enroll-hint">
+              <span class="hint-caret"></span>
+              <p class="text-helper" id="auto-enroll-helper">
+                ${_("If this option is <em>checked</em>, users who have not yet registered for {platform_name} will be automatically enrolled.").format(platform_name=settings.PLATFORM_NAME)}
+                ${_("If this option is left <em>unchecked</em>, users who have not yet registered for {platform_name} will not be enrolled, but will be allowed to enroll once they make an account.").format(platform_name=settings.PLATFORM_NAME)}
+                <br /><br />
+                ${_("Checking this box has no effect if 'Unenroll' is selected.")}
```

Comment:
Nit: I think this should be 
`Checking this box has no effect if 'Revoke' is clicked.`
It doesn't matter much because the checkbox is disabled.